spec/environment_helpers/file_helpers_spec.rb

```diff
@@ -0,0 +1,72 @@
+RSpec.describe EnvironmentHelpers::FileHelpers do
+  subject(:env) { ENV }
+
+  describe "#file_path" do
+    let(:name) { "FOO" }
+
+    context "with required: true" do
+      subject(:file_path) { ENV.file_path(name, required: true) }
+
+      context "when the environment value is not set" do
+        before { expect(ENV["FOO"]).to be_nil }
+
+        it "raises a MissingVariableError" do
+          expect { file_path }.to raise_error(EnvironmentHelpers::MissingVariableError, /not supplied/)
+        end
+      end
+
+      context "when the environment value is set" do
+        context "but it does not point to an existing file" do
+          with_env("FOO" => "bar")
+          it { is_expected.to eq Pathname.new("bar") }
+        end
+
+        context "and it points to an existing file" do
+          with_env("FOO" => __FILE__)
+          it { is_expected.to eq Pathname.new(__FILE__) }
+        end
+      end
+    end
+
+    context "without a default specified" do
+      subject(:file_path) { ENV.file_path(name) }
+
+      context "when the environment value is not set" do
+        before { expect(ENV["FOO"]).to be_nil }
+        it { is_expected.to be_nil }
+      end
+
+      context "when the environment value is set" do
+        context "to a pathname that does not exist" do
+          with_env("FOO" => "bar")
+          it { is_expected.to eq Pathname.new("bar") }
+        end
+
+        context "to a pathname that does exist" do
+          with_env("FOO" => __FILE__)
+          it { is_expected.to eq Pathname.new(__FILE__) }
```

Comment:
I think we no longer need to use `__FILE__` in the specs at all, since we're not checking file-existence. Might want to drop the test-cases around that as well, unless you think they're worth keeping just to make it clear?